scripts/docker/start.sh

```diff
@@ -15,7 +15,7 @@ rm -f tmp/pids/*.pid
 if [ "$RAILS_ENV" == "development" ]; then
   exec bundle exec foreman start
 else
-  if [ ! -d public/assets ]
+  if [ ! -d public/api-assets ] || [ "$(ls -A public/api-assets)" == "" ]
```

Comment:
I think i'd prefer we switch to building these assets in the prod / staging Dockerfile rather than on boot, we need to ensure we have short boot / deploy times before the container is active.

Reply:
If I remember correctly we tried that once, but it fails because Rails tries to load the secrets.

Reply:
I've tried this in #3329 copied from here, so seems to work, https://github.com/zooniverse/caesar/blob/8d8b5d31d65b84a47e30cea5ccd77776ca5b4549/Dockerfile#L24